### PR TITLE
adding libraries and modules using ES6 features

### PR DESCRIPTION
**Pull request by ZahraArshia**
Hello 😄 
in this project, the awesome books' webpage script file is divided into 3 modules (nav.js, operations.js and time.js) doing the same work as before which was adding and removing books and showing a list of existing books.
Also, the time update mechanism is changed using luxon library.

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi @ZahraArshia , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not working. Only the HTML template is displayed. I can't add a book to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: modules/nav.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+function homePage() {
+  addSection.style.display = 'none';
+  listSection.style.display = 'none';
+  contactSection.style.display = 'none';
+  home.style.display = 'block';
+  list.style.color = 'black';
+  addNew.style.color = 'black';
+}
```
Comment (er-contreras) [inline modules/nav.js]: - [ ] It's a requirement use arrow functions. 😉

File: modules/time.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function addTime() {
+  // eslint-disable-next-line no-undef
+  document.getElementById('time').innerHTML = luxon.DateTime.local().toLocaleString(luxon.DateTime.DATETIME_FULL_WITH_SECONDS);
+}
```
Comment (er-contreras) [inline modules/time.js]: - [ ] It's a requirement use arrow functions. 😉

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @ZahraArshia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ZahraArshia** [comment]
Hello, thanks for the review 
traditional functions changed to arrow functions

**Review by Whoistolu — APPROVED** [review]
### Hi @ZahraArshia,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**